sharedbuffers/mapped_struct.py

```diff
@@ -2802,7 +2803,7 @@ def pack_into(cls, obj, buf, offs, idmap = None, implicit_offs = 0):
                 widmap.link(objid, obj)
 
         packer = cls.PACKER
-        timestamp = int(time.mktime(obj.timetuple()))
+        timestamp = int(calendar.timegm(obj.timetuple()))
```

Comment:
This doesn't work right for datetime, but it does work for date.

I'd say revert datetime, and lets fix date for now.